idl/github.com/uber/cadence/admin.thrift

```diff
@@ -38,7 +38,15 @@ service AdminService {
     )
 }
 
-struct InquiryWorkflowExecutionResponse{
+struct InquireWorkflowExecutionRequest {
+  10: optional string domain
+  20: optional shared.WorkflowExecution execution
+}
+
+struct InquireWorkflowExecutionResponse{
   10: optional string shardId
   20: optional string historyAddr
+  30: optional string otherInfo
```

Comment:
what is this?

Reply:
@wxing1292 said he need more information in mutableStateBuilder like update/deleteActivityInfos, making sure that they are empty.

Reply:
but not called otherInfo, as a string.

moreover, shardId is a int64, i believe.

Reply:
Talked offline, I will get rid of otherInfo